ckanext/dia_theme/templates/package/search.html

```diff
@@ -1,6 +1,11 @@
 {% ckan_extends %}
 
 
+{% block links %}
+    <!-- see DATA-390 Copied from the global theme, because we do not want to include the override of the links block done by the ckanext-dcat plugin-->
```

Comment:
Drop this inline comment - add to the commit message if you need to leave a note for your future self ;)

Reply:
Changing this to a jinja comment so that future code readers can discover why we are overriding the links block without calling `super()`